Dockerfile

```diff
@@ -0,0 +1,54 @@
+ARG BUILD_DATE=""
+ARG ARCH=amd64
+ARG PYTHON_VERSION=3
+
+
+# First stage
+FROM docker.io/${ARCH}/python:${PYTHON_VERSION}-alpine3.8 as builder
+
+# Install deps
+COPY requirements.txt /tmp
+
+RUN apk add --no-cache \
+    build-base \
+    postgresql-dev \
+    linux-headers
```

Comment:
We only need ``make``, ``g++`` and ``musl-dev`` IIRC.

Reply:
The first stage need some headers. For example `linux/limits.h` is not include in musl-dev.
I think `make` and `g++` are useless.